docs/dg/architecture.md

```diff
@@ -65,9 +65,12 @@
 ## ReportGenerator(Main)
 
 [`ReportGenerator`](https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/report/ReportGenerator.java)
- 1. uses `GitClone` API to clone the repository from *GitHub*.
- 1. copies the template files into the designated output directory.
- 1. uses `CommitReporter` and `AuthorshipReporter` to produce the commit and authorship summary, respectively.
+ 1. clones repositories from *GitHub* using the `GitClone` API, in a multi-threaded fashion.
```

Comment:
I know this is not related to this PR, but is the GitClone API provided by GitHub? If not, probably no need to mention *GitHub* in this step?